composer.json

```diff
@@ -22,6 +22,7 @@
     },
   "require": {
     "composer/installers": "^1.11",
+    "litesaml/lightsaml": "^1.4",
```

Comment:
Should not be part of https://github.com/mautic/mautic/blob/5.x/app/composer.json ?  @escopecz times ago write how to add this https://mautic.slack.com/archives/CQMKV0RU1/p1645619361591259

Reply:
@kuzmany you are right, I will adjust this PR. thanks for the note

Reply:
@kuzmany @escopecz the package `lightSAML/SymfonyBridgeBundle` did not update its `composer.json` file. and there is an active PR since Jan without no response https://github.com/lightSAML/SymfonyBridgeBundle/pull/35

What we can do is fork their repo and use it as we did with oauth2

Reply:
I guess we'll have to to unblock. Or do you see an alternative path?

Reply:
Please create a fork of their repo, I do not have access to do it. I will take care of the changes

Reply:
https://github.com/mautic/lightsaml Ping me if you'll have troubles with write access.

Reply:
Oops, I created a fork of a wrong repo. Deleted now. The fork already exists: https://github.com/mautic/SymfonyBridgeBundle

@fedys is looking into this.

Reply:
@escopecz yes, I was the who forked `SymfonyBridgeBundle`, I was about to do this PR but I got into many things until now. 

@fedys if you want to finish this PR, please feel free. if not I will do it

Reply:
At some point this bundle will break the upgrade process, I was trying to upgrade to `5.4` to use `rector` but I got this issue, so it is a good time to resolve this issue as well before it happens
``` 
   - symfony/dependency-injection[v4.0.0-BETA1, ..., v4.4.8] require php ^7.1.3 -> your php version (8.0; overridden via config.platform, actual: 8.0.26) does not satisfy that requirement.
    - lightsaml/symfony-bridge 1.3.0 requires symfony/dependency-injection ~2.7|~3.0|~4.0 -> satisfiable by symfony/dependency-injection[v2.7.0-BETA1, ..., 2.8.x-dev, v3.0.0-BETA1, ..., 3.4.x-dev, v4.0.0-BETA1, ..., 4.4.x-dev].
    
```

Reply:
@mabumusa1 I will take care of it